python/paddle/fluid/executor.py

```diff
@@ -901,6 +911,11 @@ def _run_impl(self, program, feed, fetch_list, feed_var_name,
                 use_program_cache=use_program_cache)
 
         program._compile(scope, self.place)
+
+        if _has_lod_tensor_array(program._program, fetch_list):
+            raise ValueError("Currently, The type of item in fetch_list should be" \
```

Comment:
“Currently, The type of item in fetch_list should be LoDTensor the program is compiled”

Did you miss "when" ?

It is better to be “Currently, the type of item in fetch_list should be LoDTensor when using CompiledProgram”